### PR TITLE
Getting started readme

### PR DESCRIPTION
**Pull request by NickGowdy**
When I first forked the repo, I wasn't sure how to run the codebase locally. I figured it out by looking at `.vscode/launch.json` and seeing the command it runs inside VS code.

I think it would be better overall if we have some examples of how to run the code locally for everybody as not all devs use VS Code. 

I've also added a contributing section as we didn't have one. However it's very basic and I'm not sure how much detail we should include.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,15 @@ Documentation on how to use and extend Ebiten UI is available at [ebitenui.githu
 
 ![Screenshots](ebiten-ui.gif)
 
+Quick Start
+------
+Ebiten UI is written in Go 1.19 which is available at [https://go.dev/](https://go.dev/).
+
+There are Ebiten UI examples that can be found in the `_examples/` folder. Here are some example of how to run them locally:
```
Comment (mcarpenter622) [inline README.md:26]: This is great, but can you please add a small section about how these commands must be run from the root directory of the project else they won't work?

File: README.md [hunk 2]
```diff
@@ -43,3 +52,7 @@ Maintainers
 -------
 * Currently maintained by Mark Carpenter <mark@bird-mtn.dev>
 * Originally written by Maik Schreiber <blizzy@blizzy.de>
+
+Contributing
+-------
+Want to help develop Ebiten UI? Check out our [current issues](https://github.com/ebitenui/ebitenui/issues). Want to know the steps on how to start contributing, take a look at the [open source guide](https://opensource.guide/).
```
Comment (mcarpenter622) [inline README.md:58]: What do you think about updating the link for the open source guide to : https://opensource.guide/how-to-contribute/

**Review by mcarpenter622 — CHANGES_REQUESTED** [review]
Hey Nick, Thank you for the PR. I have a couple requests for ya, and can you please sign the CLA from the bot =)

**Comment by mcarpenter622** [comment]
Could you also please add a section about running the examples as a wasm:
From root:
go run github.com/hajimehoshi/wasmserve@latest ./_examples/widget_demos/textinput

Open browser to localhost:8080

**Comment by mcarpenter622** [comment]
Hey @NickGowdy are you going to update this? Let me know if not and I'll clean this PR up to get it in and closed.

**Comment by NickGowdy** [comment]
> Hey Nick, Thank you for the PR. I have a couple requests for ya, and can you please sign the CLA from the bot =)

Hey @mcarpenter622 , I've signed the CLA multiple times and it hasn't changed the workflow approval message. Is there something I'm missing?

**Comment by mcarpenter622** [comment]
I am not really sure? Regardless of the bot, if you make the requested changes I can merge the change. I did update it a little while back to a new version which invalidated everyone's previous acceptance. It may be something related to that

**Comment by mcarpenter622** [comment]
@NickGowdy  Since it's been a while, I went ahead and incorporated the ideas from this PR. I am going to close this PR for now. Thank you for your contribution!

**Pull request closed without merging**